source/features.html.erb

```diff
@@ -0,0 +1,63 @@
+  <nav class="breadcrumbs" aria-label="Breadcrumbs">
+      <ol itemscope itemtype="http://schema.org/BreadcrumbList">
+          <li class="breadcrumbs__item" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+              <a href="https://www.gov.uk/service-toolkit#components" itemprop="item"><span itemprop="name">Components</span></a>
+          </li>
+          <li class="breadcrumbs__item breadcrumbs__item" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+              <a href="/" itemprop="item"><span itemprop="name">GOV.UK Pay</span></a>
+          </li>
+          <li class="breadcrumbs__item breadcrumbs__item--active" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+              <a href="#main" itemprop="item"><span itemprop="name">Features and roadmap</span></a>
+          </li>
+      </ol>
+  </nav>
+<div class="container">
+  <div class="grid-row">
+    <div class="column-one-third">
+      <nav class="sub-navigation">
+        <ol itemscope itemtype="http://schema.org/ItemList">
+          <li class="sub-navigation__item sub-navigation__item--active" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+            <a href="/features.html" itemprop="item"><span itemprop="name">Features</span></a>
+          </li>
+          <li class="sub-navigation__item" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+            <a href="roadmap.html" itemprop="item"><span itemprop="name">Roadmap</span></a>
+          </li>
+        </ol>
+      </nav>
+    </div>
+
+      <div class="column-two-thirds">
+              <h1 class="heading-large">Top features</h1>
+                  <p>GOV.UK Pay uses established Payment Service Providers to deliver your payments service.</p>
```

Comment:
Tim confirms this is deliberate, usually people just write PSPs but that's too unclear.